README.md

```diff
@@ -392,7 +392,7 @@ Contributions are welcome. Please take a quick look at the [contribution guideli
  * [OAuth2](https://crystal-lang.org/api/OAuth2.html) - OAuth2 client (Crystal stdlib)
  * [OpenSSL](https://crystal-lang.org/api/OpenSSL.html) - bindings to libssl (Crystal stdlib)
  * [Socks](https://github.com/wontruefree/socks) - SOCKS proxy
- * [transfer_more](https://github.com/Nephos/transfer_more) - Clone of transfer.sh to uploads files
+ * [transfer_more](https://git.sceptique.eu/Sceptique/transfer_more) - Clone of transfer.sh to uploads files
```

Comment:
Does this service have integration with CI tools?

Reply:
No

Reply:
Could you suggest than how we can control whether your shard still works or not?

We usually request periodic CI build to be able to do that since Crystal often releases breaking changes

Reply:
It is not my repository, I was updating the link and noticing incomplete travis. I believe @Nephos is still the maintainer.

Reply:
Ah, thanks. It is a duplicate of #383